sdk/dotnet/Dagger.SDK/GraphQL/Types.cs

```diff
@@ -0,0 +1,83 @@
+using System.Text;
+
+namespace Dagger.SDK.GraphQL;
+
+public abstract class GraphQLType
```

Comment:
I'm not sure which word is better between `Type` and `Value`. `Value` sound reasonable because we use it for put the value.